content/articles/verifying-domain-resolution.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Verifying Domain Resolution
+excerpt: This article explains the different ways you can verify that a domain is resolving.
+categories:
+- Domains
+---
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+# Verifying Domain Resolution
+
+Certain changes to your domain require that its resolution be properly configured. To ensure this, we run checks to verify that your domain is resolving correctly before you make any changes. If we detect anything that might prevent a change from taking effect, a notification will appear listing the failed verifications and the actions you need to take to get your domain resolving.
```

Comment:
This intro feels a little too heavy. I wonder if we could introduce it in a little lighter, layman way:

```suggestion
When you make a change to your domain, you expect it to take effect. Unfortunately, there are some scenarios where, for one reason or another, your changes won't apply to the "real world".

To mitigate this, we run a series of verifications to ensure that the change will take affect as expected – *before any change is applied to your domain's DNS zone*. If we detect any issues, a list of failed verifications will appear – along with the actions you need to take to get your domain resolving.
```

WDYT, @itsalyse?

Reply:
```suggestion
When you make a change to your domain, you expect it to take effect. Unfortunately, there are some scenarios where, for one reason or another, your changes fail to apply.

To mitigate this, we run a series of verifications *before any change is applied to your domain's DNS zone* to ensure that the change will take effect as expected. If we detect any issues, a list of failed verifications will appear, along with the actions you need to take for your domain to resolve.
```